packages/postgres-database/src/simcore_postgres_database/models/clusters.py

```diff
@@ -0,0 +1,97 @@
+from enum import Enum
+
+import sqlalchemy as sa
+from sqlalchemy.sql import func
+
+from .base import metadata
+
+# FIXME: Needs some endpoint/credentials to access the cluster
+
+
+class ClusterType(Enum):
+    AWS = "AWS"
+    ON_PREMISE = "ON_PREMISE"
+
+
+clusters = sa.Table(
+    "clusters",
+    metadata,
+    sa.Column(
+        "id",
+        sa.BigInteger,
+        nullable=False,
+        primary_key=True,
+        doc="Clusters index",
+    ),
+    sa.Column("name", sa.String, nullable=False, doc="Display name"),
+    sa.Column("description", sa.String, nullable=True, doc="Short description"),
+    sa.Column(
+        "type",
+        sa.Enum(ClusterType),
+        nullable=False,
+        doc="Classification of the cluster",
+    ),
+    sa.Column(
+        "owner",
+        sa.BigInteger,
+        sa.ForeignKey(
+            "groups.gid",
+            name="fk_services_meta_data_gid_groups",
+            onupdate="CASCADE",
+            ondelete="RESTRICT",
+        ),
+        nullable=False,
+        doc="Identifier of the group that owns this cluster",
+    ),
+    sa.Column(
+        "created",
```

Comment:
MINOR: I thought all columns related to datetime should add ``_at`` suffix by convention, i.e. ``created_at`` and ``modified_at``

Reply:
after discussion added an issue for this. we'll unify to something without ```_at``` as this seems to be the majority! 👍